src/pages/PollWorkerScreen.test.tsx

```diff
@@ -25,12 +26,14 @@ it('renders PollWorkerScreen', async () => {
   const { getByText } = render(
     <PollWorkerScreen
       appMode={VxMarkOnly}
+      appPrecinctId={defaultPrecinctId}
       ballotsPrintedCount={0}
       election={electionSampleWithSeal as Election}
       isPollsOpen
       isLiveMode={false}
       machineId="1"
       printer={new NullPrinter()}
+      tally={[]}
```

Comment:
should this be an `EmptyTally` abstraction? Feels like you might be breaking the abstraction here by specifying an empty array directly.

Reply:
👍 Fixed.